components/policy-mgt/org.wso2.carbon.policy.mgt.core/src/main/java/org/wso2/carbon/policy/mgt/core/mgt/impl/PolicyManagerImpl.java

```diff
@@ -719,9 +719,9 @@ public void addAppliedPolicyFeaturesToDevice(DeviceIdentifier deviceIdentifier,
             boolean exist = policyDAO.checkPolicyAvailable(deviceId);
```

Comment:
should be meaningful to have a name like "policyExistsForDevice" rather than having just "exist".